### PR TITLE
[feedback needed] attempt to improve class comment for RemoteString

### PR DESCRIPTION
**Pull request by hogoww**
I don't think that's enough. I'd love some feedback on this

### CODE REVIEW

**Review by MarcusDenker — APPROVED** [review]

File: src/System-Sources/RemoteString.class.st [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 
-I can return either a String or a Text.  Some a chunk is followed by a second chunk (beginning with ]style[) containing style information.  The encoding is like this:
+I can return either a String or a Text. A chunck of code is followed by a second chunk (beginning with ]style[) containing style information. The encoding is like this:
 
 max: aNumber
 	^ self > aNumber ifTrue: [self] ifFalse: [aNumber]!
 ]style[(14 50 312)f1,f1b,f1LInteger +;i!
 
-Allowed TextAttributes are TextFontChange, TextEmphasis, TextColor, TextDoIt, TextKern, TextLink, TextURL.  TextFontReference and TextAnchor are not supported.
+Allowed TextAttributes are TextFontChange, TextEmphasis, TextColor, TextDoIt, TextKern, TextLink, TextURL.
```
Comment (myroslavarm) [inline src/System-Sources/RemoteString.class.st:19]: I would say "TextAttributes that are allowed are: ..."

File: src/System-Sources/RemoteString.class.st [hunk 2]
```diff
@@ -1,18 +1,19 @@
 "
 My instances provide an external file reference to a piece of text.  It may be the sourceCode of a method, or the class comments of a class.
 
-The changes file or file-in file usually has a chunk that is just the source string of a method:
+The 'changes' file or 'file-in' file usually has a chunk that is just the source string of a method:
```
Comment (myroslavarm) [inline src/System-Sources/RemoteString.class.st]: I'm not sure whether the changes and file-in files are meant to be different here. If they are, i would say "changes and file-in files usually have..."; if they aren't, i would say "the changes (or file-in) file usually has..."

File: src/System-Sources/RemoteString.class.st [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
 
 max: aNumber
 	^ self > aNumber ifTrue: [self] ifFalse: [aNumber]!
 
-I can return either a String or a Text.  Some a chunk is followed by a second chunk (beginning with ]style[) containing style information.  The encoding is like this:
+I can return either a String or a Text. A chunck of code is followed by a second chunk (beginning with ]style[) containing style information. The encoding is like this:
```
Comment (guillep) [inline src/System-Sources/RemoteString.class.st]: Not all chunks have style. Only some of them.

File: src/System-Sources/RemoteString.class.st (hunk 2 above)
Comment (guillep) [inline src/System-Sources/RemoteString.class.st]: They -> have ;)

**Review by guillep — CHANGES_REQUESTED** [review]
Hi @hogoww could you fix the couple of typos? Otherwise I think it is good :)

**Comment by hogoww** [comment]
Don't know why i cannot answer you on the conversation.
The typo is fixed.
The other one i'm not sure I understand the initial text. so i'm having trouble correcting it.
I'll see tomorrow with a rested mind.

File: src/System-Sources/RemoteString.class.st (hunk 3 above)
Comment (guillep) [reply src/System-Sources/RemoteString.class.st]: What this "wanted to mean" (:P) was that in a source file (sources, changes, fileouts) we store chunks of code. Some chunks are styled (they have parts in bold, or font color...). When a chunk is styled, it is followed by a second chunk (prefixed with something like ]sytle[), and specifying the style.

**Comment by hogoww** [comment]
I actually used what you wrote because i think this was waaaaaaaaaay better, I hope you don't mind.
Thanks ! =)

**Comment by guillep** [comment]
Not at all :P

I wrote that with a baby that was drooling on my arm and moving around, so if that was good enough...

**Comment by hogoww** [comment]
Guess he inspired you :D

**Review by guillep — APPROVED** [review]
Good enough. I'll integrate

**Comment by guillep** [comment]
Thanks :)

**Pull request merged**